dist/tools/ci/changed_files.sh

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Kaspar Schleiser <kaspar@schleiser.de>
+# Copyright 2014 Ludwig Knüpfer <ludwig.knuepfer@fu-berlin.de>
+# Copyright 2014 Hinnerk van Bruinehsen <h.v.bruinehsen@fu-berlin.de>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+changed_files() {
+    : ${FILEREGEX:='\.([CcHh]|[ch]pp)$'}
```

Comment:
it's like a NOP in bash. It is used when you want the side effects of, for example, `${FILEREGEX:='\.([CcHh]|[ch]pp)$'}`, which is to set `FILEREGEX = '\.([CcHh]|[ch]pp)$'` if it was not already set before.